servant-client-core/src/Servant/Client/Core/ResponseUnrender.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE ApplicativeDo #-}
+module Servant.Client.Core.ResponseUnrender where
```

Comment:
The name is a bit general. Maybe it could be prefixed by MultiVerb, or the class could be named something like `MultiResponseUnrender`?

Reply:
Done